spec/the-f-library-fsharpcoredll.md

```diff
@@ -296,6 +296,10 @@ type 'T option =
 
 See also the [Language Guide](https://learn.microsoft.com/dotnet/fsharp/language-reference/options) and the [FSharp.Core documentation](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-option-1.html).
 
+## Choices and Results
+
+Check the [Language Guide](https://learn.microsoft.com/dotnet/fsharp/language-reference/results) and the [FSharp.Core documentation](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core.html#category-3_4).
```

Comment:
Shouldn't we still have some examples here as in the previous section? Seems a bit weird for a language spec to just refer you to some other documents.

Reply:
> Seems a bit weird for a language spec to just refer you to some other documents.

True. The existing F# spec is in many aspects not optimal. But this current effort is not about improving the spec, but about integrating the RFCs of the last years. That's why I have followed the style of the rest of this chapter (which is really more an appendix than a full part of the spec).

Reply:
Got it.